boards/arduino_mkrwifi1010/README.md

```diff
@@ -0,0 +1,26 @@
+# Adafruit Gemma M0 Board Support Crate
+
+This crate provides a type-safe API for working with the [Adafruit Gemma M0
+board](https://www.adafruit.com/product/3501).
```

Comment:
The title and product link/text need to be updated here please.

Reply:
Done!